components/social/org.wso2.carbon.social.sql/src/main/java/org/wso2/carbon/social/sql/SQLActivityPublisher.java

```diff
@@ -676,4 +682,57 @@ public int warmUpRatingCache(String targetId)
         }
     }
 
+	public JsonObject updateActivity(JsonObject activity) throws SocialActivityException {
```

Comment:
Thanks @thushara35 , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/e7ac8ba67c76149caa910a2f0a4c6bb7ac3757ea)